README.md

```diff
@@ -1 +1,64 @@
-# hello-microverse
\ No newline at end of file
+
+# Hello Microverse
+
+>This is my first Project.
+
+
+## Built With
+
+- Major languages
+- Frameworks
+- Technologies used
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
+## Authors
```

Comment:
- [OPTIONAL] kindly change `Authors ` to `Author` as you are the only author of this project.